docs/sources/release-notes/v2-4.md

```diff
@@ -49,6 +49,11 @@ We have enabled the [in-memory FIFO cache by default](https://github.com/grafana
 
 ## Bug fixes
 
+### 2.4.2 bug fixes
+2.4.2 fixes these important bugs:
+
+* ADD BUG FIX HERE
```

Comment:
nit (non-blocker): do we need `ADD BUG FIX HERE` here?. Ok if you plan to remove in the next PR when you add actual bug fixes.